module-app/src/main/java/edu/flab/election/dto/ElectionFindOrderByTotalVotedCntDto.java

```diff
@@ -0,0 +1,16 @@
+package edu.flab.election.dto;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class ElectionFindOrderByTotalVotedCntDto {
+	long offset;
```

Comment:
```suggestion
	long countOffset;
```

좀더 명확한 변수명이면 좋지 않을까 생각되네요 ㅎ